packages/gui/src/components/nfts/NFTContextualActions.tsx

```diff
@@ -573,6 +577,55 @@ function NFTInvalidateContextualAction(props: NFTInvalidateContextualActionProps
   );
 }
 
+/* ========================================================================== */
+/*                          "Add To"/ "Remove from" Filter NFT                */
+/* ========================================================================== */
+
+type NFTAddToGalleryContextualActionProps = NFTContextualActionProps;
+
+function NFTAddToGalleryContextualAction(props: NFTAddToGalleryContextualActionProps) {
+  const { selection } = props;
+
+  const openDialog = useOpenDialog();
+  const disabled = selection?.items.length === 0;
+  const [userFoldersNFTs, setUserFoldersNFTs] = useLocalStorage('user-folders-nfts', {});
```

Comment:
usePrefs instead of useLocalStorage
user can move his configuration file to another computer